### ISSUE TITLE
Potential for version 1.2?

### ISSUE THREAD

**Issue by adlkas** [nathanfarlow/PineappleCAS#22]
I would like to know if the project is still being worked on and if so when the approximate date of release of v. 1.2 will be.

**Comment by nathanfarlow** [comment]
To be honest, there is not any active development on this project. Currently the only feature I can think of to add to version 1.2 would be https://github.com/nathanfarlow/PineappleCAS/issues/12. While we're at it we could also try compiling with newer versions of the [toolchain](https://github.com/CE-Programming/toolchain) have released to see if it improves the speed at all.

**Comment by adriweb** [comment]
Compiling with LLVM does greatly improve the speed, I did benchmarks a few months ago :)

**Comment by arusher999** [comment]
Haha dude I have an Algebra 1 final exam coming up and I think this PCAS thing is so going to help me pass lol. There are a few issues but its still super awesome. Now I'm just waiting for someone to make Doom for the ti 84 plus ce.

**Comment by FPSUsername** [comment]
It would be great if [partial derivatives](https://www.symbolab.com/solver/partial-derivative-calculator) could be added as well.

**Comment by arusher999** [comment]
I've been waiting for half a year... I don't think that it is going to happen..

**Comment by adriweb** [comment]
Maybe someone can try this version built with the latest LLVM-based toolchain: [PCAS_llvm_build.zip](https://github.com/nathanfarlow/PineappleCAS/files/7004142/PCAS_llvm_build.zip)


You may need the latest libs from the toolchain master branch as of this date: https://github.com/CE-Programming/toolchain/suites/3502016106/artifacts/83462478

**Comment by arusher999** [comment]
I will try it.

**Comment by adriweb** [comment]
OK thanks. Yeah might be some leftovers in memory, but that should be completely harmless. You can clear after that.

**Comment by nathanfarlow** [comment]
@FPSUsername @arusher999 Partial derivatives are already a part of PCAS. You can choose which variable to take the derivative with respect to. @adriweb Thanks for compiling with the new toolchain! That's pretty awesome. Those artifacts on the home screen also appear at times with the old toolchain, so nothing to worry about.

**Comment by FPSUsername** [comment]
Oh yes, that's right.

**Comment by arusher999** [comment]
@nathanfarlow Can we think about adding #12 to the feature list for PineappleCAS? The new toolchain does seem to be faster, though.

**Comment by UInt2048** [comment]
For future visitors, you can get the latest C libraries (clibs) here: https://github.com/CE-Programming/libraries/releases

**Comment by UInt2048** [comment]
@nathanfarlow If we have differentiation, how hard would it be to add some integration support?

**Comment by adriweb** [comment]
Extremely hard. Differentiation is easy, integration is completely different and way harder in general, requiring a lot more algorithms. See Risch, for instance.
In 3 words: not gonna happen :p

**Comment by arusher999** [comment]
How about = equations, like 5x=40 and stuff like that but with fractions?

**Comment by StandingPadAnimations** [comment]
Just curious, is there any interest in a v1. 2?

**Comment by arusher999** [comment]
> Just curious, is there any interest in a v1. 2?

I do not believe that there is any active development on this project currently. Quite a shame since this has the potential to become an actually full CAS for the CE.
